app-modules/engagement/resources/views/components/message-center-content.blade.php

```diff
@@ -46,7 +46,7 @@ class="max-h-full overflow-y-auto"
                 id="message-center-content"
             >
                 <div
-                    class="sticky top-0 z-[5] flex h-12 w-full items-center justify-between bg-gray-100 px-4 dark:bg-gray-700">
+                    class="sticky top-0 z-[5] flex h-12 w-full items-center justify-between bg-gray-100 px-4 underline dark:bg-gray-700">
                     <h1 class="ml-2 flex">
                         <a
                             class="hover:underline"
```

Comment:
We can probably remove this class now since the whole thing is always underlined.